Automattic/jetpack
# Stats: Add stats-admin package

#### Changes proposed in this Pull Request:
The `stats` package contains the essential functionalities (Loading the pixel and some endpoints) for Stats so that Stats could easily be integrated with Jetpack the plugin, the Search plugin and etc.

Recently we are refreshing the Jetpack Stats experience, mostly the Stats Dashboard. The PR introduces another package `stats-admin`, which would handle everything happening from the WP-Admin side. We create it as a separate package because the admin part isn't always needed by plugins and would otherwise bloat size of the plugin.

The package adds route that forwards requests to WPCOM and loads assets from `https://widgets.wp.com/calypso-stats/`.

Calypso Stats: https://github.com/Automattic/wp-calypso/pull/69231

#### Other information:


#### Jetpack product discussion
pejTkB-1h-p2

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Apply `D92140-code` to your sandbox
* Add `x.x.x.x widgets.wp.com` to your hosts file <- please change it to your sandbox IP
* Checkout and build current branch `add/stats-admin-package` on your local test site
* Open `/wp-admin/admin.php?page=stats&calypso_stats=1` on you local test site
* Ensure Jetpack Stats looks reasonable and **_mostly_** works
* Ensure legacy Stats still works
* Switch WordPress language to French (other languages fine, but make sure Calypso supports it)
* Open `/wp-admin/admin.php?page=stats&calypso_stats=1` on you local test site
* Ensure the page is translated

<img width="813" alt="image" src="https://user-images.githubusercontent.com/1425433/201553915-bdbf798a-e291-4de9-be7d-b5663d9d6c9d.png">
